src/material-experimental/mdc-slider/slider.ts

```diff
@@ -155,7 +155,7 @@ export class MatSliderThumb implements AfterViewInit {
    * instead be capped at either the default min or max.
    *
    */
-  private _initializeInputMinMax(): void {
+  private _initializeInputMinMaxAndStep(): void {
```

Comment:
Maybe just call this `_initializeInputState` so the name doesn't have to be updated any time we add/remove a property?